fea-rs/src/compile/feature_writer.rs

```diff
@@ -86,11 +90,40 @@ impl<'a> FeatureBuilder<'a> {
     /// I have no idea how best to approximate that in this API :/
     pub fn define_mark_class(
         &mut self,
-        class_name: impl Into<SmolStr>,
-        members: Vec<(GlyphClass, Option<AnchorTable>)>,
+        members: impl Into<GlyphClass>,
+        anchors: HashMap<NormalizedLocation, (i16, i16)>,
+    ) {
```

Comment:
@cmyr I wonder if something in this direction could work?

Reply:
a mark class can have members that have different anchor positions.

Reply:
resolving as I will update per vc